app/logic/volunteers.py

```diff
@@ -89,13 +89,13 @@ def setProgramManager(username, program_id, action):
     '''
     adds and removes the studentstaff from program that makes them student manager.
 
-    param: uername - a string
+    param: username - a string
            program_id - id
            action: add, remove
 
     '''
     studentstaff=User.get(User.username==username)
-    if action == "add" and studentstaff.isCeltsStudentStaff==True:
+    if action == "add":
         studentstaff.addProgramManager(program_id)
```

Comment:
Considering you removed the need for them to be a student staff, studentstaff is not a great variable name.